CMSgov/mint-app
# [MINT-2501] Provider physician List Modal

# MINT-2501

## Description

- Created `ProviderAndSupplierModal`
- Add translations for said modal
- Add button to open up modal in the Participant and Provider page 
- 

## How to test this change

1.  Navigate to Participant and Provider page 1 and add `Medicare providers/suppliers` to the first question
2. Verify that link is there
3. Click link and verify that modal opens 
4. Verify that modal functions correctly/as expected
5. Test the side nav
6. Test filter
7. test pagination
8. verify all matches figma

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
